test/base/TextTest.ooc

```diff
@@ -226,6 +226,17 @@ TextTest: class extends Fixture {
 			text = t"%s %s %i %i" format("test", "number", 0, 1)
 			expect(text == t"test number 0 1")
 		})
+		this add("implicit toText", func {
+			one := t"123" + 456 + 7.89f
```

Comment:
Except `one := (t"123" toFloat() + 456 + 7.89f) toText()` would give you something like `one := t"586.89"`...
I do like to have the option of writing like this, instead of a myriad of `toText()`/`toString()` calls.